packages/wrangler/pages/functions/template-worker.ts

```diff
@@ -88,9 +88,11 @@ function* executeRequest(request: Request, env: Env) {
     }
   }
 
-  // Finally, yield to the asset-server
+  // Finally, yield to the fallback service (`env.ASSETS.fetch` in Pages' case)
   return {
-    handler: () => env.ASSETS.fetch(request.url, request),
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-expect-error we don't define a type for the special global __FALLBACK_SERVICE_FETCH__ since we replace it when we're building anyway
```